WhatsApp/firestorerepository/src/main/java/com/example/firestorerepository/repositories/contacts/ContactRepositoryImpl.kt

```diff
@@ -13,7 +13,15 @@ class ContactRepositoryImpl @Inject constructor() : ContactRepository {
         return Observable.create<List<Contact>> { emitter ->
             val data = mutableListOf(
                 Contact("1"),
-                Contact("2")
+                Contact("2"),
```

Comment:
Without looking on how the code works, what are this magic values for?

Reply:
this is just test data until we set up firebase